pkg/interface/src/views/apps/publish/components/Note.tsx

```diff
@@ -45,7 +45,7 @@ export function Note(props: NoteProps & RouteComponentProps) {
 
   const comments = note?.comments || [];
   const file = note?.file;
-  const newfile = file ? file.slice(file.indexOf(";>") + 2) : "";
+  const newfile = file ? file.slice(file.indexOf(";>") + 3) : "";
```

Comment:
Imagine a world where the fix in EditPost is applied and people start using it in their posts, no longer having the newline at the start. Won't this then cut off the first character, if we do this in both places?

Reply:
Good point